pkg/block/mem/adapter.go

```diff
@@ -3,19 +3,20 @@ package mem
 import (
 	"bytes"
 	"context"
-	"crypto/sha256"
+	"crypto/md5" //nolint:gosec
```

Comment:
So why the change?

But if you do need to, place a comment above the line explaining why this is cool ("... same as S3... used only in test..." etc.)

Reply:
Will add next to the func that calls this package it will make more sense there.